main.js

```diff
@@ -52,6 +51,14 @@ function createNewRuler(windowInfo) {
 }
 
 /**
+ * Toggle the visibility of the Center Guides in the active ruler
+ */
+function toggleCenterGuidesCommand() {
+	var activeRuler = BrowserWindow.getFocusedWindow();
+	activeRuler.send('toggle-center-guides');
```

Comment:
Let's make sure `activeRuler` isn't undefined before calling `send`. Otherwise an error is thrown when no ruler is selected (for example when creating a ruler, clicking on another application, then going on the menu and clicking on Toggle Center Guides)